v1 of creators cart front end PR

saisumanthkacham opened the pull request:
pages added- cart ,wishlist, product display, product page, authentication

feat- state management, sorting and filtering

saisumanthkacham committed:
intial commit

pages added- cart ,wishlist, product display, product page, authentication

feat- state management, sorting and filtering

saisumanthkacham committed:
code refactor

saisumanthkacham committed:
code refactor

omkarborude commented on src/App.jsx:
```diff
@@ -0,0 +1,68 @@
+
+import './index.css';
+import {ProductPage} from "./pages/productsPage/productPage.jsx"
```
You can import all components from one File!

saisumanthkacham replied:
yeah did that now .

omkarborude commented on src/pages/data.js:
```diff
@@ -0,0 +1,52 @@
+import faker, { seed } from 'faker';
+
+// import { useEffect,useState } from "react";
+// import axios from 'axios'
```
commented Code!

omkarborude commented on src/pages/data.js:
```diff
@@ -0,0 +1,52 @@
+// }
+
+// export const data= FetchData()
+// console.log("outside the func",data)
```
same here!

saisumanthkacham replied:
actually, I accidentally pushed the code I'm working on so I commented them, anyhow I  fixed it.

omkarborude reviewed (approved):
looks good to me!

saisumanthkacham committed:
wired the api server and front end

kunwar-11 commented on src/App.jsx:
```diff
@@ -0,0 +1,93 @@
+
+
+
+//  useEffect(()=>{
+
+//   (async ()=> {
+//     console.log("entered the function")
+//     try{
+//       const {data}=await axios.get("https://ecom-api.sumanth5234.repl.co/data")
+//       dispatch({type:"GET-DATA-FROM-SERVER",payLoad:data})
+//     console.log("inside the func",data)
+//     }
+//     catch(err){
+//       console.log(err)
+//     }
+   
+// })()
+
+//  },[])
+  
```
Remove unnecessary comments

saisumanthkacham replied:
actually, I accidentally pushed the code I'm working on so I commented them, anyhow I  fixed it.

kunwar-11 reviewed (approved):
Looks good to me 
Just remove unnecessary comments

saisumanthkacham committed:
code refactor

saisumanthkacham committed:
added API calls for cart

added api calls for  ADD-TO-CART functionality

saisumanthkacham committed:
added api calls from client side

saisumanthkacham committed:
code refactor

saisumanthkacham committed:
navigate added

added navigate in useEffect , to divert to homepage if logged in , instead of going to auth page

saisumanthkacham committed:
style: css added

css added to every component

saisumanthkacham committed:
style: aside width updated

saisumanthkacham committed:
style: margin updated

saisumanthkacham committed:
style : refactored

saisumanthkacham committed:
style: code refactor

home page image and text properties updated

saisumanthkacham committed:
style: home image prop refactored

saisumanthkacham committed:
style: refactor

saisumanthkacham committed:
style: margin refactor

saisumanthkacham committed:
text refactored

saisumanthkacham committed:
style: footer icons color updated

saisumanthkacham committed:
add: added toasts

saisumanthkacham committed:
code refactor

saisumanthkacham committed:
edit: github link edited

saisumanthkacham committed:
add: test credentials and  auth toasts

saisumanthkacham committed:
style : plus minus

cursor property added to inc and dec buttons

saisumanthkacham committed:
add: add to cart button

saisumanthkacham committed:
code refactor

saisumanthkacham committed:
refactor
